frontend/components/Cell.js

```diff
@@ -117,16 +120,129 @@ export const Cell = ({
     const class_code_folded = code_folded && cm_forced_focus == null
 
     let show_input = errored || class_code_differs || !class_code_folded
+    const prepareFileBase64 = (file) =>
+        new Promise((resolve, reject) => {
+            const { name, type } = file
+            const fr = new FileReader()
+            fr.onerror = () => reject("Failed to read file!")
+            fr.onloadstart = () => {}
+            fr.onprogress = ({ loaded, total }) => {}
+            fr.onload = () => {}
+            fr.onloadend = ({ target: { result } }) => resolve({ fileBase64: String(result).replace(/.*base64,/, ""), name, type })
+            fr.readAsDataURL(file)
+        })
 
+    const uploadAndCreateCodeTemplate = async (file) => {
+        if (!(file instanceof File)) return " #  File can't be read"
+        setSavingFile(true)
+
+        const {
+            message: { success, file_path, file_name },
+        } = await prepareFileBase64(file).then(
+            (preparedObj) => {
+                return requests.write_file(cell_id, preparedObj)
+            },
+            () => alert("Pluto can't save this file 😥")
+        )
+        setSavingFile(false)
+        if (!success) {
+            alert("Pluto can't save this file 😥")
+            return "# File save failed"
+        }
+
+        const [name, extension] = file_name.split(".").map((a) => a.replace(/["\-,#@!\%\s+\;()$&*\[\]\{\}']/g, ""))
+        switch (file?.type) {
+            // https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
+            case "text/plain":
+                if (extension === "csv")
+                    return `begin
+    using TableIO, DataFrames #
+    df_${name} = DataFrame(read_table(raw"${file_path}"); copycols=false)
+end`
+                if (extension === "txt")
+                    return `begin
+    file = open(raw"${file_path}")
+    txt_${name} = read(file, String)
+end`
+            case "application/vnd.ms-excel":
+            case "application/vnd.oasis.opendocument.spreadsheet":
+            case "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet":
+                return `begin
+    using TableIO, DataFrames
+    df_${name} = DataFrame(read_table(raw"${file_path}",
+    #, "Sheet1"  # Uncomment This if you need to read a specific sheet
```

Comment:
remove leading `,`

Suggestion:
`#"Sheet1"  # Uncomment This if you need to read a specific sheet`